admin/tests/acceptance/certification-center-form_test.js

```diff
@@ -31,17 +30,21 @@ module('Acceptance | Certification-center Form', function (hooks) {
     });
 
     // when
-    await visit('/certification-centers/new');
+    const screen = await visitScreen('/certification-centers/new');
+
     await fillInByLabel('Nom du centre', name);
-    await fillIn('#certificationCenterTypeSelector', type.value);
+    await fillInByLabel("Type d'établissement", type.value);
     await fillInByLabel('Identifiant externe', externalId);
-    await clickByLabel('Pix+ Droit');
-    await clickByLabel('Ajouter');
+    await click(screen.getByRole('checkbox', { name: /Pix\+ Droit/ }));
+    await click(screen.getByRole('button', { name: /Ajouter/ }));
```

Comment:
```suggestion
    await fillIn(screen.getByRole('textbox', { name: 'Nom du centre' }), name);
    await fillIn(screen.getByRole('textbox', { name: "Type d'établissement" }), type.value);
    await fillIn(screen.getByRole('textbox', { name: 'Identifiant externe' }), externalId);
    await click(screen.getByRole('checkbox', { name: 'Pix+ Droit' }));
    await click(screen.getByRole('button', { name: 'Ajouter' }));
```
1. Utiliser la testing library uniquement pour être cohérent
2. Utiliser plutôt une string pour l'option `name` s'il n'y a pas besoin d'un regex

Reply:
Yes, vu ensemble, c'est ok 👍